php/php.api.phpmodule/src/org/netbeans/modules/php/api/PhpVersion.java

```diff
@@ -84,7 +85,12 @@
      * PHP 7.4.
      * @since 2.65
      */
-    PHP_74(Bundle.PhpVersion_PHP_74());
+    PHP_74(Bundle.PhpVersion_PHP_74()),
+    /**
+     * PHP 7.4.
+     * @since 2.74
+     */
+    PHP_80(Bundle.PhpVersion_PHP_80());
```

Comment:
I didn't increment the spec version because I would like to avoid conflicting. Probably, the version is 2.74(2.74 + 2) when the merge window is opened. So, I fixed only `@since`.
current spec version:
https://github.com/apache/netbeans/blob/eb15c28aa6ee25d5e9c673ad04c0f21c7079b501/php/php.api.phpmodule/manifest.mf#L4